core/src/main/java/org/elasticsearch/search/aggregations/bucket/nested/NestedAggregator.java

```diff
@@ -44,28 +44,27 @@
 import java.util.List;
 import java.util.Map;
 
-/**
- *
- */
 public class NestedAggregator extends SingleBucketAggregator {
 
     static final ParseField PATH_FIELD = new ParseField("path");
 
-    private BitSetProducer parentFilter;
+    private final BitSetProducer parentFilter;
     private final Query childFilter;
 
     private DocIdSetIterator childDocs;
     private BitSet parentDocs;
```

Comment:
It looks like we can remove these instance variables and only declare them in `getLeafCollector`?